### ISSUE TITLE
summon-aws-secrets formula has broken download URL

### ISSUE THREAD

**Issue by tabacco** [cyberark/homebrew-tools#49]
## Summary

The summon-homebrew-formula formula has a broken download link. The download link refers to the summon repo, not the summon-aws-secrets repo. Otherwise, the URL looks correct.

## Steps to Reproduce
```
brew tap cyberark/tools
brew install summon-aws-secrets
```

## Expected Results

Installs summon-aws-secrets

## Actual Results

```
==> Downloading https://github.com/cyberark/summon/releases/download/v0.4.1/summon-aws-secrets-darwin-arm64.tar.gz
curl: (22) The requested URL returned error: 404

Error: summon-aws-secrets: Failed to download resource "summon-aws-secrets"
Download failed: https://github.com/cyberark/summon/releases/download/v0.4.1/summon-aws-secrets-darwin-arm64.tar.gz
```

## Reproducible

   * [x] Always 
   * [ ] Sometimes
   * [ ] Non-Reproducible
   
## Version/Tag number

Attempting to install latest (0.4.1)

## Environment setup

n/a

## Additional Information

Problematic lines:
https://github.com/cyberark/homebrew-tools/blob/master/summon-aws-secrets.rb#L12
https://github.com/cyberark/homebrew-tools/blob/master/summon-aws-secrets.rb#L21
https://github.com/cyberark/homebrew-tools/blob/master/summon-aws-secrets.rb#L33

I believe the fix is as easy as updating the URL to refer to cyberark/summon-aws-secrets instead of cyberark/summon. I'd open a PR but this looks like it's generated by goreleaser

**Comment by john-odonnell** [comment]
Thanks for the report, @tabacco.
Fixed this in [summon-aws-secrets#61](https://github.com/cyberark/summon-aws-secrets/pull/61) and [homebrew-tools#46](https://github.com/cyberark/homebrew-tools/pull/46). The brew tap should be updated - I was able to successfully install on my Macbook - let me know if you're unblocked and we can close this issue.

**Comment by tabacco** [comment]
Looking good, thanks!
